mw-webapp/.eslintrc

```diff
@@ -29,6 +29,7 @@
         ]
       }
     ],
+    "no-restricted-exports": ["error", {"restrictedNamedExports": ["default"]}],
     "capitalized-comments": [
```

Comment:
why not "no-restricted-exports": ["error", {"restrictDefaultExports": [direct]"}]
Can you please explain fifference bwtween these 2 rules

Reply:
i think they are interchangable as per doc. Changed to "no-restricted-exports": ["error", {"restrictDefaultExports": [direct]"}] anyway.